test-toolkit/integration/local/test_distributed_training.py

```diff
@@ -32,10 +32,10 @@ def fixture_dist_gpu_backend(request):
 
 
 @pytest.mark.skip_gpu
-def test_dist_operations_path_cpu(docker_image, dist_cpu_backend, sagemaker_local_session, tmpdir):
+def test_dist_operations_path_cpu(ecr_image, dist_cpu_backend, sagemaker_local_session, tmpdir):
```

Comment:
does the image _have_ to be in ECR for this test? if not, I'd lean toward not having "ecr" in the fixture name

Reply:
I found it confusing to have ecr and none ecr image names.
Also i think if the image is not present locally our local mode can pull image only from ecr.

I guess ecr is a bad name. 
What do you think about having one fixture for image name, e.g.: image_uri that would handle building image_uri ecr like if have everything we need and just regular like if we don't?

Reply:
`image_uri` sounds good